test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/CreateModerationResponseResult.cs

```diff
@@ -28,8 +28,8 @@ internal CreateResult(bool flagged, CreateCategories categories, CreateCategoryS
         /// <summary> Whether the content violates [OpenAI's usage policies](/policies/usage-policies). </summary>
         public bool Flagged { get; }
         /// <summary> A list of the categories, and whether they are flagged or not. </summary>
-        public CreateCategories Categories { get; }
+        public CreateModerationResponseResultCategories Categories { get; }
```

Comment:
This is anonymous model: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/moderation/models.tsp#L32